search/domain/base.py

```diff
@@ -8,6 +8,9 @@
 
 from mypy_extensions import TypedDict
 
+from search import consts
+
+
 EASTERN = timezone("US/Eastern")
```

Comment:
minor: this can be moved into the new `search.consts`.

Reply:
Done.